MDAnalysis/distopia
# Add googletest for testing

* Added googletest
* change CMake to use proper options for building tests and timings
* Added a dummy test file
* make test should now work
